ludwig/modules/convolutional_modules.py

```diff
@@ -392,14 +482,16 @@ def call(self, inputs, training=None, mask=None):
                 'and pool.'
             )
 
-        return hidden
+        return hidden  # (batch_size, len(parallel_layers) * out_channels, seq_size)
```

Comment:
this should be batch, seq_length, parallel_layers * out_channels

Reply:
corrected comment on both inputs and output hidden shape